plantcv/plantcv/image_fusion.py

```diff
@@ -0,0 +1,79 @@
+# Fuse two images
+
+import os
+import numpy as np
+from plantcv.plantcv import fatal_error
+from plantcv.plantcv import Spectral_data
+from plantcv.plantcv import params
+from plantcv.plantcv.hyperspectral.read_data import _make_pseudo_rgb
+from plantcv.plantcv._debug import _debug
+
+
+def image_fusion(img1, img2, wvs1, wvs2, array_type = None, filename = None):
```

Comment:
```suggestion
def image_fusion(img1, img2, wvs1, wvs2, array_type=None, filename=None):
```

No spaces between keywords and default values